otterdog/jetty.jsonnet

```diff
@@ -310,7 +310,8 @@ orgs.newOrg('jetty') {
         actions_can_approve_pull_request_reviews: false,
       },
     },
-    orgs.newRepo('jetty.test.policy') {
+    orgs.newRepo('jetty-test-policy') {
+      aliases: ["jetty.test.policy"],
```

Comment:
Is this entry something we have to keep permanently in the jetty.jsonnet now?

Reply:
I am wondering that as well, for example if we moved i-jetty over from jetty-project and then decided to rename it in the jetty org, would that disrupt follows and things. Or is the alias something that protects that?